src/entities/transactions/transaction.ts

```diff
@@ -60,6 +61,7 @@ export interface NewTransaction {
   nonce: number | null;
   protocol?: ProtocolType | null;
   flashbots?: boolean;
+  ensRegistrationName?: string;
```

Comment:
added these to handle ens registration commit tx speed ups from the speed up screen